apps/frontend/src/onboarding/onboarding.css

```
@@ -0,0 +1,243 @@
+.ob-root {
+  --ob-gap: clamp(1rem, 6.4vh, 5rem);
+  /* The agreement's body is tall enough that the standard gap pushes it into
+     the footer buttons. */
+  --ob-gap-tight: clamp(0.5rem, 2.4vh, 1.6rem);
+  /* Separates the eyebrow from the headline and the footer from the body. */
+  --ob-band-gap: clamp(1.5rem, 3.4vh, 3rem);
+  --ob-pad-top: clamp(0.85rem, 3vh, 2.6rem);
+  --ob-pad-bottom: clamp(2.5rem, 6.7vh, 5.5rem);
+  --ob-progress-bottom: clamp(1rem, 3.1vh, 2.6rem);
+  font-family: var(--site-font-body);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)